src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -999,58 +998,53 @@ public int GetCookiesCount()
             int count = 0;
             lock (SyncRoot)
             {
-                foreach (CookieCollection cc in _list.Values)
+                foreach (KeyValuePair<string, CookieCollection> pair in _list)
                 {
+                    CookieCollection cc = pair.Value;
                     count += cc.Count;
                 }
             }
             return count;
         }
 
-        public ICollection Values
```

Comment:
Right.  But it was public (on an internal class).  Hence my questioning whether this was dead code (after your changes).  I take it the answer is "yes" ;)